travis/install_llvm.sh

```diff
@@ -0,0 +1,14 @@
+export LLVM_PATH=${HOME}/llvm/clang+llvm-${LLVM_VERSION}-x86_64-linux-gnu-ubuntu-14.04
```

Comment:
Let's put this in the `scripts` directory, just to keep the root of the repo cleaner. We should add git pre and post hooks to their also at some point...